bench_test.go

```diff
@@ -11,32 +11,48 @@ import (
 
 func BenchmarkBase58EncodeToString(b *testing.B) {
 	data := make([]byte, 8192)
+	data[0] = 0xff // without this, it's just skipping zero bytes
 	b.SetBytes(int64(len(data)))
 	for i := 0; i < b.N; i++ {
-		basen.Base58.EncodeToString(data)
+		x := basen.Base58.EncodeToString(data)
+		_ = x
```

Comment:
This sounds like what you're referring to: https://github.com/golang/go/issues/5790